tests/data/with_methods.puml

```diff
@@ -0,0 +1,19 @@
+@startuml
+class tests.modules.withmethods.withmethods.Coordinates {
+  x: float
+  y: float
+}
+class tests.modules.withmethods.withmethods.Point {
+  PI: float {static}
+  coordinates: Coordinates
+  day_unit: TimeUnit
+  hour_unit: TimeUnit
+  time_resolution: Tuple[str, TimeUnit]
+  x: int
+  y: str
+  void __init__(x: int, y: str, unit: str, u: float, z: List[int])
+  (float, float) get_coordinates()
+  {static} Point from_values(x: int, y: str, u: float, z: List[int])
```

Comment:
Since py2puml produces UML class diagrams I'd rather stick to the original UML syntax with return type declared before the signature. But I kept the Python return type hinting syntax in the next commit to leave this discussion opened.

Reply:
I kind of agree with you, let's follow the UML convention. Using the Python type annotation syntax here was an artefact of using `signature=str(signature)` (see https://github.com/lucsorel/py2puml/pull/43/files#diff-adfc82ab9429d3c5fa93391a8044c10a783c3c77df333a23f8a5eaf31168f91fR87), but I didn't like relying on the `str` representation of a signature, which can change over time :+1:

Reply:
I reverted the expected results in the test cases to the UML convention in my latest commit .